internal/versions/versions.go

```diff
@@ -308,10 +314,11 @@ func (h *handler) AddReleaseImage(releaseImageUrl, pullSecret, ocpReleaseVersion
 		}
 	}
 
-	// Ensure a relevant OsImage exists
+	// Ensure a relevant OsImage exists. For multiarch we disabling the code below because we don't know yet
+	// what is going to be the architecture of InfraEnv and Agent.
 	osImage, err := h.GetOsImage(ocpReleaseVersion, cpuArchitecture)
-	if err != nil || osImage.URL == nil {
-		return nil, errors.Errorf("No OS images are available for version: %s", ocpReleaseVersion)
+	if cpuArchitecture != common.MultiCPUArchitecture && (err != nil || osImage.URL == nil) {
```

Comment:
So we can actually move the 'cpuArchitecture != common.MultiCPUArchitecture' check to line 319. I.e. no need to even try GetOsImage in case of MultiCPUArchitecture.

Reply:
Yes, good suggestion 👍🏻